django/publicmapping/redistricting/management/commands/setup.py

```diff
@@ -723,7 +722,7 @@ def create_template(self, config):
                 plan_name,
                 legislative_body.id,
                 path,
-                owner=admin,
+                owner_id=admin.pk,
```

Comment:
This breaks the processing in `index2plan`, which is still expecting an `owner`, rather than an `owner_id` -- do you think it's appropriate to carry this change through to celery in this PR, or to make another story and fix it there?

Reply:
I think I've made the necessary changes to `index2plan` in this PR; is there something I've missed? Regardless, that was definitely my intent, so if there's anything else that needs to be done I can add it.

Reply:
Hmm, lemme rebuild containers and see if that magic fixes it...

Reply:
Ok yeah thanks rebuilt containers -- now getting an email about not being able to send emails, which, yeah, DistrictBuilder, I _know_